--- atanks-6.6/atanks.desktop.orig	2023-12-26 08:23:15.959229156 +0100
+++ atanks-6.6/atanks.desktop	2023-12-26 09:11:07.917003759 +0100
@@ -2,8 +2,10 @@
 Name=Atomic Tanks
 Comment=Destroy your opponents' tanks.
 Comment[es]=Destruye los tanques de tus oponentes.
+Comment[pl]=Zniszcz wrogie czołgi
 Type=Application
 Exec=atanks
 Icon=atanks
 Categories=Game;ArcadeGame;
 Terminal=false
+# vi: encoding=utf-8
